cockatrice/src/window_main.cpp

```diff
@@ -107,12 +107,15 @@ void MainWindow::processConnectionClosedEvent(const Event_ConnectionClosed &even
         case Event_ConnectionClosed::LOGGEDINELSEWERE: reasonStr = tr("You have been logged out due to logging in at another location."); break;
         default: reasonStr = QString::fromStdString(event.reason_str());
     }
-    QMessageBox::critical(this, tr("Connection closed"), tr("The server has terminated your connection.\nReason: %1").arg(reasonStr));
+    QMessageBox::critical(this, tr("Connection closed"), tr("The server has terminated your connection.") + QString("<br>")
+                                                        +tr ("Reason:") + QString(" %1").arg(reasonStr));
 }
 
 void MainWindow::processServerShutdownEvent(const Event_ServerShutdown &event)
 {
-    serverShutdownMessageBox.setInformativeText(tr("The server is going to be restarted in %n minute(s).\nAll running games will be lost.\nReason for shutdown: %1", "", event.minutes()).arg(QString::fromStdString(event.reason())));
+    serverShutdownMessageBox.setInformativeText(tr("The server is going to be restarted. All running games will be lost.") + QString("<br><br>")
+                                                + tr("Reason for shutdown:") + QString(" %1<br>").arg(QString::fromStdString(event.reason()))
+                                                + tr("Time until shutdown:") + QString(" %n ", "", event.minutes()) + tr("minutes"));
```

Comment:
[QObject::tr()](http://doc.qt.io/qt-5/qobject.html#tr) supports disambiguation strings ("") and different strings to handle plurals (%n). You can just use %1 in your refactoring.